### PR TITLE
Human readable color names for the GIMP palette

### PR DESCRIPTION
**Pull request by vanillajonathan**
Closes issue #26.

### CODE REVIEW

File: colors/build.js [hunk 1]
```diff
@@ -54,10 +54,11 @@ Name: Firefox/Photon
 
 `],
     'formatter': (color, variant, value) => {
+      color = color.charAt(0).toUpperCase() + color.slice(1);;
```
Comment (bwinton) [inline colors/build.js]: double-semicolon here…

**Comment by aminalhazwani** [comment]
Cool, thank you @vanillajonathan! Do you have any idea on how to set the palette name?

**Review by bwinton — APPROVED** [review]
Other than that nit, it looks good to me!

**Comment by vanillajonathan** [comment]
@aminalhazwani The palette shows up as "Firefox/Photon" in GIMP for me.
As per the "Name" key-value pair defined on the second line in the `colors.gpl` file.

Version: GIMP 2.8.16 (Windows 10)

image

**Comment by aminalhazwani** [comment]
Perfect, thanks @vanillajonathan!

**Pull request merged**